web/pdf_page_view.js

```diff
@@ -109,6 +109,8 @@ class PDFPageView {
     regularAnnotations: true,
   };
 
+  #previousRotation = 0;
```

Comment:
Nit: We've been pretty good about ordering private fields alphabetically, so can we please move this up a little bit?

Reply:
Yep, sure.
Do you know if could add a linter for doing that automagically ?